### PR TITLE
chore: add a checkbox for documentation in PR template

### PR DESCRIPTION
**Pull request by grignards**
Everything is in the title.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- Manual, quality tests not necessary here.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 - [ ] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
 - [ ] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
 - [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
-
+- [ ] If there are modifications in Tracim configuration (`development.ini`), make sure it is documented in `backend/doc/setting.md`
```
Comment (PhilippeAccorsi) [inline .github/pull_request_template.md:12]:
```suggestion
- [ ] If there are modifications in Tracim configuration (file like `development.ini`), make sure it is documented. Ex: new parameter in `development.ini` need to be added in `backend/doc/setting.md`
```

Comment (grignards) [reply .github/pull_request_template.md]: Updated.

**Review by PhilippeAccorsi — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply .github/pull_request_template.md]: thanks ;)

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 - [ ] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
 - [ ] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
 - [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
-
+- [ ] If there are modifications in the Tracim configuration files (eg. `development.ini`), make sure it is documented in `backend/doc/setting.md`
```
Comment (raphj) [inline .github/pull_request_template.md:12]:
```suggestion
- [ ] If there are modifications in the Tracim configuration files (eg. `development.ini`), make sure it is documented in `backend/doc/setting.md` and relevant people are aware of them
```

**Review by raphj — CHANGES_REQUESTED** [review]

**Review by raphj — APPROVED** [review]

**Pull request merged**